static/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "Static Drupalcamp 2018",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.html",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
Is this normal?

Reply:
This is generated by default when runnning npm init.

Reply:
Done